code/game/supplyshuttle.dm

```diff
@@ -46,6 +46,192 @@ var/datum/controller/supply/supply_controller = new()
 
 //SUPPLY PACKS MOVED TO /code/defines/obj/supplypacks.dm
 
+/obj/docking_port/stationary/supply
```

Comment:
I would recommend reworking this. Have /obj/docking_port/stationary/supply as your base class - with your width/height etc in place and have your in game, placed stationary docks as subclasses. You could have /Almayer and /offsite - for example - as subclasses.

You want to do this because the sizes make your base class really simple. I don't know how exactly but you could have the offsite just being in infinite transit, so that you don't actually need to place anything. I made a similar request in https://github.com/cmss13-devs/cmss13/pull/2824